Yoast/javascript
# Improve the stemming of all the forms of the entries in exception lists

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Improve the stemming of all the forms of the entries in the exception lists

## Test instructions
This PR can be tested by following these steps:

* Add other forms of the exception lists entries to the determineSpec, for example from removeSuffixFromFullForms list. e.g. benefietje, benefietjes
* run yarn test and make sure there is no failing test. 


## Quality assurance


Fixes #
